layout/container.ftd

```diff
@@ -67,14 +69,14 @@ link: index.html
 -- ds.rendered:
```

Comment:
Why are you using `ds.rendered` here? `ds.rendered` is used to show source code and output of that source code. If you only want to show source code, use `ds.code`.